NEWS.md

```diff
@@ -51,6 +51,7 @@ Standard library changes
   `keep` that are to be kept as they are. ([#38597]).
 * `getindex` can now be used on `NamedTuple`s with multiple values ([#38878])
 * `keys(::RegexMatch)` is now defined to return the capture's keys, by name if named, or by index if not ([#37299]).
+* `keys(::Generator)` is now defined to return the iterator's keys ([#34678](https://github.com/JuliaLang/julia/pull/34678))
```

Comment:
```suggestion
* `keys(::Generator)` is now defined to return the iterator's keys ([#34678])
```
The urls will be automatically generated

Reply:
(you can also use `[ci skip]` in the commit message or PR title to skip CI for changes like this)

Reply:
if they would I wouldn't write it that way 😛 
![image](https://user-images.githubusercontent.com/5960902/108626818-78566400-7452-11eb-9406-4235901b7c8a.png)

Reply:
They get resolved by a script in contrib that is run before each release

Reply:
but that means that the most time that people look into it they will be annoyed for having to find the right issue or pr themselves :/ I mean, I would guess that most people that look into the news file will do so at least once before the release. Why don't we just have both? Adding compatibility for that to the script should be easy. I'd even take on that if it's written in Julia. 🙈